server/graphql_schemas/event/schema.py

```diff
@@ -232,7 +234,7 @@ def mutate_and_get_payload(cls, root, info, **input):
         try:
             receiver = AndelaUserProfile.objects.get(
                 user__email=receiver_email)
-            event = Event.objects.get(id=from_global_id(event_id)[1])
+            event = Event.objects.get(id=event_id)
```

Comment:
I am not sure why we changed this, but to the best of my knowledge, we should be sending/receiving global ids (like _RXZlbnROb2RlOjQ0_) to and from the client. This is why we have the `from_global_id` helper function from `graphql_relay` to resolve the expected global ids to actual database ids. Is that pattern still valid here?

Reply:
Yea true. Had issues testing the mutation with `Insomnia GraphQL Playground`, will change it back.